_pubs/explanation-alignment.md

```diff
@@ -0,0 +1,21 @@
+---
+title: 'Explanation Alignment: Quantifying the Correctness of Model Reasoning At Scale'
+authors:
+  - key: hbang
+  - key: aboggust
+  - key: arvindsatya
+venue: eXCV
```

Comment:
Could we add, alongside `venue`, a top-level YAML key-pair of `type: workshop` to indicate that this is a workshop paper.